tests/factories.py

```diff
@@ -38,6 +38,7 @@ class Meta:
     postal_code = factory.Faker("postalcode")
     shipping_price = round(random.uniform(1.00, 100.00), 2)
     date_created = FuzzyDate(date(2008, 1, 1))
+    status = factory.Faker("status")
```

Comment:
Again didn't know how to tell factory generator that it should be either empty or "Cancelled"

Reply:
like this:

https://github.com/nyu-devops/sample-accounts/blob/4ee57777a446dbc2740d6f90027dcf93205ca628/tests/factories.py#L60